SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchPreview/EventSearchAssetVoltageDisturbances.tsx

```diff
@@ -25,53 +25,66 @@ import React from 'react';
 import SEBrowserService from './../../../../TS/Services/SEBrowser';
 import moment from 'moment';
 import { SEBrowser } from '../../../global';
+import Table from '@gpa-gemstone/react-table';
+
+const seBrowserService = new SEBrowserService();
+
+interface IDisturbanceData {
+    EventType: string;
+    Phase: string;
+    PerUnitMagnitude: number;
+    DurationSeconds: number;
+    StartTime: string;
+    SeverityCode: string;
+    IsWorstDisturbance: boolean;
+}
 
 const EventSearchAssetVoltageDisturbances: React.FC<SEBrowser.IWidget<any>> = (props) => {
-    const [tableRows, setTableRows] = React.useState<JSX.Element[]>([]);
-    const seBrowserService = new SEBrowserService();
+    const [data, setData] = React.useState<IDisturbanceData[]>([]);
 
     React.useEffect(() => {
-        if (props.eventID >= 0) createTableRows(props.eventID);
-        return () => { };
+        if (props.eventID >= 0) loadDisturbancesData(props.eventID);
     }, [props.eventID]);
 
-    const createTableRows = (eventID: number) => {
-        seBrowserService.getEventSearchAsssetVoltageDisturbancesData(eventID).done((data) => {
-            const rows = data.map((d, i) => {
-                const style = { backgroundColor: d.IsWorstDisturbance ? 'lightyellow' : 'transparent' };
-                return (
-                    <tr key={i} style={style}>
-                        <td>{d.EventType}</td>
-                        <td>{d.Phase}</td>
-                        <td>{(d.PerUnitMagnitude * 100).toFixed(1)}</td>
-                        <td>{(d.DurationSeconds * 1000).toFixed(2)}</td>
-                        <td>{moment(d.StartTime).format('HH:mm:ss.SSS')}</td>
-                    </tr>
-                );
-            });
-            setTableRows(rows);
+    const loadDisturbancesData = (eventID: number) => {
+        seBrowserService.getEventSearchAsssetVoltageDisturbancesData(eventID).done((response) => {
+            const parsedData = response.map((d) => ({
+                ...d,
+                PerUnitMagnitude: (d.PerUnitMagnitude * 100).toFixed(1),
+                DurationSeconds: (d.DurationSeconds * 1000).toFixed(2),
+                StartTime: moment(d.StartTime).format('HH:mm:ss.SSS'),
+            }));
+            setData(parsedData);
```

Comment:
Because you removed `createTableRows` the `style` piece got lost. You can add that exact styling to the `Table` via `prop.selected={(r) => r.isworsDisturbance}`